.claude/skills/component-flow-guide/details/verification-checklist.md

```
@@ -0,0 +1,55 @@
+- [ ] 문서가 실제 API와 일치하는가?
+- [ ] 예제가 동작하는가?
+- [ ] Storybook 스토리가 테마별로 정상인가?
+- [ ] `bun generate:registry` 실행했는가?
+- [ ] 타입 에러가 없는가? (`bun typecheck`)
+- [ ] Visual Test 통과했는가? (Agent Browser)
+
+## 흔한 실수
+
+### 잘못된 순서
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)